res/zee-worker.js

```diff
@@ -25,7 +25,7 @@ var Ca=[ic,Kb];var Da=[jc,Nb,Lb,Cb,Mb,jc,jc,jc];var Ea=[kc,Db];var Fa=[lc,Xa,Ya,
     compress: function (data, compressionLevel) {
       var arrayData = (typeof data === 'string') ? (new TextEncoder()).encode(data) : data;
       var mode = (compressionLevel === undefined) ? 'wb' : 'wb' + compressionLevel;
-      var gzModule = createGzModule(arrayData.length * 2.5);
+      var gzModule = createGzModule(arrayData.length * 5);
```

Comment:
Isn't the issue that the problematic profile is very small (110KB), and so the calculation doesn't handle properly the fixed costs ?

Other profiles typically are several MBs, some can even be 50MB, and I'm afraid that a simple multiplicator is not the good approach here.

How about keeping a 2.5 multiplicator, but with a minimum of 1MB ? `Math.max(arrayData.length * 2.5, 1024 * 1024)`